frameworks/g-cloud-9/questions/services/standardsCSASTAR.yml

```diff
@@ -2,7 +2,7 @@ name: CSA STAR certification
 question: >
   Do you have a current CSA Security, Trust & Assurance Registry (STAR) certification that covers the security of
   your service?
-
+filter_label: CSA STAR
```

Comment:
It's in the current draft like this, I expect this all to change.

Acronyms are interesting, arguably if you want to filter on it, you know the acronym, but filters we think are particularly valuable, we clearly need to explain, somehow. Karl's thinking about it anyway.